x-pack/metricbeat/tests/system/config/libbeat.yml.j2

```diff
@@ -0,0 +1,96 @@
+#================================ General =====================================
```

Comment:
This file is copy-pasted from https://github.com/elastic/beats/blob/master/metricbeat/tests/system/config/metricbeat.yml.j2

Reply:
We should parametrize the [template loader](https://github.com/elastic/beats/blob/master/libbeat/tests/system/beat/beat.py#L301) in a different way so we can reuse these files.
I am ok with duplicating these files by now to not block this.